Sources/RxFeedback/UIView+Extensions.swift

```diff
@@ -62,6 +62,15 @@ extension UI {
             })
         }
     }
+    
+    /**
+     Bi-directional binding of a system State to UI and UI into Events,
+     Strongify element.
+     */
+    public static func bind<State, Event, ElementType>(_ element: ElementType, _ bindings: @escaping (ElementType, Observable<State>) -> (Bindings<Event>))
```

Comment:
I think `ElementType` to `WeakOwner` and `_ element: ElementType` -> `_ owner: WeakOwner` would be more descriptive. `ElementType` doesn't document a lot :)